legate/util/system.py

```diff
@@ -86,6 +86,13 @@ def LIB_PATH(self) -> str:
     def cpus(self) -> tuple[CPUInfo, ...]:
         """A list of CPUs on the system."""
 
+        def expand_range(value: str) -> tuple[int, ...]:
+            if "-" not in value:
+                return tuple((int(value),))
+            start, stop = value.split("-")
+
+            return tuple(x for x in range(int(start), int(stop) + 1))
```

Comment:
can just call `tuple` on the `range` directly, no need for a generator
```suggestion
            return tuple(range(int(start), int(stop) + 1))
```

Reply:
Also `expand_range` doesn't really need to be a closure I don't think. It could go in `legate.util` and then it would be possible to add plain pytest unit tests for it.

Reply:
Done. Added test_expand_range.